daffodil-core/src/main/scala/org/apache/daffodil/dsom/IIBase.scala

```diff
@@ -179,7 +179,17 @@ abstract class IIBase(xml: Node, xsdArg: XMLSchemaDocument, val seenBefore: IIMa
     res
   }.value
 
-  final lazy val schemaLocationProperty = getAttributeOption("schemaLocation")
+  final lazy val schemaLocationProperty = {
+    val prop = getAttributeOption("schemaLocation")
+    prop.map { text =>
+      if (text.contains("edu/illinois/ncsa/daffodil")) {
+        SDW("schemaLocation property uses deprecated edu/illinois/ncsa/daffodil path instead of org/apache/daffodil. Converting to new path.")
+        text.replace("edu/illinois/ncsa/daffodil", "org/apache/daffodil")
+      } else {
+        text
+      }
+    }
+  }
```

Comment:
Supports backwards compatibility for schemas that still reference the old paths. For example, it is common to have an import with schemaLocation="edu/illinois/ncsa/daffodil/built-in-formats.xsd"